docs/release/trg-7/trg-7-06.md

```diff
@@ -31,19 +31,40 @@ The information can be provided in a static way or you can link directly to the
 
 :::info
 
-The links to your repostitory have to reference the tagged version (*) the component delivery has been created / build from!
+The links to your repository have to reference the tagged version (*) the component delivery has been created / build from!
 
 :::
 
 Example:
 
     [Your product name](URL to the YOUR repository)
     * License: Apache-2.0
-    * Licence Path: <URL to the Apache License in your repository (*)>
+    * License Path: <URL to the Apache License in your repository (*)>
     * NOTICE: <URL to the NOTICE file in your repository (*)>
     * Source URL: <URL to the your repository (*)>
     * Commit ID: <the commit id the component delivery has been created / build from>
 
+In order to align with the overall style guide, it's recommended to use the About card provided in the [Shared UI Components](https://www.npmjs.com/package/cx-portal-shared-components) npm library maintained by the portal product (please note that the About card is only available in this library from version 1.5.0-RC1 onwards).
+
+![About card](@site/static/img/about-card.png)
+
+Regarding placement of this card in frontend apps: for reference, this About card has been integrated in a About page which is reachable from the footer of the portal.
+
+Regarding usage of this About card, it's also recommended to add a json file to your frontend project to be imported for the real content.
+As some of the content can only be determined in the build pipeline, you could add a json file as reference containing placeholder values, e.g.:
+
+    {
```

Comment:
haven't checked it but might it make sense to add markdown code blocks around this or is it already properly formatted through the indent?

Reply:
Thanks for the hint! Unfortunately, I don't think it makes sense in this context.